### ISSUE TITLE
Inconsistent strand error

### ISSUE THREAD

**Issue by clavia96** [genometools/genometools#927]
## Problem description
I am getting the error below when i run ltr digest
gt ltrdigest: error: inconsistent strands encountered in `LTR_retrotransposon' feature in file sorted_genome.final.fa.pass.list.gff3, line 23753: found +, expected -

## Exact command line call triggering the problem

```
gt suffixerator -dna -db genome.final.fa -tis -suf -lcp -des -ssp -sds -lossless
gt gff3 -sort genome.final.fa.pass.list.gff3 > sorted_genome.final.fa.pass.list.gff3
gt -j 10 ltrdigest -hmms Pfam-A.hmm -aaout yes -outfileprefix genome.final.fa.pass.list -seqfile genome.final.fa -matchdescstart sorted_genome.final.fa.pass.list.gff3 >
 genome.final.fa.pass.list.digest.gff3
```

## What GenomeTools version are you reporting an issue for (as output by `gt -version`)?
genometools/1.5.10

**Comment by satta** [comment]
Could you probably share an example GFF3 input and sequence to reproduce this issue please?

**Comment by clavia96** [comment]
Below is the GFF3 and sequence example input

GFF3
[genome_final_gff3.txt](https://github.com/genometools/genometools/files/4042776/genome_final_gff3.txt)

Sequence
[Genome_final_fasta.txt](https://github.com/genometools/genometools/files/4042783/Genome_final_fasta.txt)

**Comment by satta** [comment]
Hi @clavia96,  I have finally found the time to take a closer look at this. The problem is that your input annotation (not from _LTRharvest_ but `LTR_retriever`) already has a strand assignment set. In the problematic case that strand contradicts the strand assignment made by LTRdigest (on the basis of the protein pHMM matches) for the `LTR_retrotransposon`, which is forward strand, versus the `LTR_retriever` assignment of reverse strand.  LTRdigest assumes that it has full control over the strands and complains if there are strand mismatches.

I am wondering how you can know the strand only knowing the positions of the LTRs and TSDs? I would expect that you would need some kind of directional feature to determine an orientation?

To address the question at hand, I would be OK with relaxing that check to simply print a warning when there are conflicting strand assignments and to leave the final strand resolution to the user. What do you think?

**Comment by clavia96** [comment]
That would be really helpful, will run it again and let you know how it goes.

Thanks for making out time to look into this!

**Comment by satta** [comment]
The change is merged in #937 -- if you want to build from git master you can try it right now!
